boards/cc1312-launchpad/dist/cc1312r1f3_gdb.conf

```diff
@@ -1 +1 @@
-target remote localhost:3333
+target extended-remote :3333
```

Comment:
When to use `target remote` and when to use `target extended-remote`?
Use of both can be found in RIOT.

Reply:
As far I know (by reverse engineering `gdb_agent_server`) `extended-remote` enables monitor commands for the debug probes that support it. Mainly the `monitor`&nbsp;`halt`/`reset` commands. The only other board that in RIOT uses extended-remote is the `cc2538dk`.

With the outdated file, executing the `target extended-remote` gave errors (not supported)